src/css/story.css

```
@@ -39,52 +39,52 @@ section#our-story::before {
     justify-content: center;
     align-items: top;
     width: 100%;
-    gap: 50px;
-    max-width: 600px;
+    gap: 3.125rem;
+    max-width: 37.5rem;
 }
 
 .story-image img {
-  width: 100%;
-  min-width: 200px;
-  object-fit: cover;
-  object-position: center;
+    width: 100%;
+    min-width: 12.5rem;
+    object-fit: cover;
+    object-position: center;
 }
 
 .story-image img.big-story-image {
```

```suggestion
.big-story-image {
```
In theorie zou dit voldoende moeten zijn, tenzij je op meerdere plekken deze class gebruikt.